### PR TITLE
fix: allow for deno.jsonc to be processed

### LINKED ISSUE #18: feat: Add support for deno.jsonc files
https://github.com/jasonjgardner/fresh_tailwind/blob/66f10f544bb5f8a480334518750ddd4074ccd47c/cli.ts#L195

https://github.com/jasonjgardner/fresh_tailwind/blob/66f10f544bb5f8a480334518750ddd4074ccd47c/cli.ts#L213

### PR DESCRIPTION
**Pull request by king8fisher**
Attempts to fix #18

### CODE REVIEW

**Comment by king8fisher** [comment]
Oops found a little bug. Fixing it

File: cli.ts [hunk 1]
```diff
@@ -192,53 +195,68 @@ async function download(root?: string, dest = "./bin"): Promise<string> {
...
     }
 
     if (!denoJson.tasks["tailwind:watch"]) {
       denoJson.tasks["tailwind:watch"] =
-        "./bin/tailwindcss -i ./src/style.css -o ./static/style.css --config ./tailwind.config.ts --watch";
+        `./bin/tailwindcss -i ${DEFAULT_STYLE_SRC} -o ${DEFAULT_STYLE_DEST} --config ./tailwind.config.ts --watch`;
     }
 
-    await Deno.writeTextFile(denoJsonPath, JSON.stringify(denoJson, null, 2));
+    // FIXME: Using JSON.stringify will remove comments from deno.jsonc. The current workaround is to use JSON.stringify to always create a deno.json file
```
Comment (jasonjgardner) [inline cli.ts:254]: Writing _back_ to .jsonc files is currently an issue. `JSONC.stringify` does not exist, so the `JSON.stringify` function would end up stripping out the comments. My current solution is to just write to a deno.json anyway, and expect devs to copy that into their deno.jsonc file.

**Review by jasonjgardner — APPROVED** [review]
I removed the usage of `existSync` to avoid race conditions

**Comment by king8fisher** [comment]
Thank you!

It's really hard to decide the best workaround with json stringify here until `JSONC.stringify` gets implemented (which would require the parser to parse the comments as well) . Fresh itself simply issues the jsonc file through the JSON emitter. So may be a warning that comments could have been stripped might be issued, as another way to deal with this situation. Plus I am pretty sure trailing commas are also removed.

**Pull request merged**